src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -383,6 +383,10 @@
         - name: Get the details of a build task, displaying the results in a table.
           text: >
             az acr build-task show -n MyBuildTask -r MyRegistry -o table
+
+        - name: Get the details of a build task including secrets.
```

Comment:
>secrets [](start = 58, length = 7)

please rename this to --secure-properties

Reply:
`--with-secure-properties` sounds good, although it's a bit lengthy, how about `--with-secret`?

@xiadu94 Did you know if anyone else is using `--secret` or something similar?